demosplan/DemosPlanCoreBundle/DataFixtures/ORM/TestData/LoadProcedureData.php

```diff
@@ -928,6 +929,13 @@ protected function loadMasterBluePrintFileElements(
             Elements::FILE_TYPE_VERTEILER,
             Elements::FILE_TYPE_NIEDERSCHRIFT_SONSTIGE,
             Elements::FILE_TYPE_SCOPING_PAPIER,
+            // added elements in bobhh master-template
```

Comment:
Better remove project name here

Reply:
done in 2f3653284b4040509cd6998474849bc911f7ec37

within the archiv: da70afc94d0b51b0d732b041d5b70b0e43b9b58f